packages/console/src/assets/docs/fragments/_redirect-uris-web.mdx

```diff
@@ -13,6 +13,6 @@ Now, let's configure your redirect URI. E.g. {`${props.defaultUri ?? 'http://loc
 
 <UriInputField name="redirectUris" />
 
-Just like signing in, users should be redirected to Logto for signing out of the shared session. Once finished, it would be great to redirect the user back to your website. For example, add `http://localhost:3000` as the post sign-out redirect URI below.
+Just like signing in, users should be redirected to Logto for signing out of the shared session. Once finished, it would be great to redirect the user back to your website. For example, add {`${props.defaultLogoutUri ?? 'http://localhost:3000'}`} as the post sign-out redirect URI below.
```

Comment:
```suggestion
Just like signing in, users should be redirected to Logto for signing out of the shared session. Once finished, it would be great to redirect the user back to your website. For example, add {`${props.defaultSignOutUri ?? 'http://localhost:3000'}`} as the post sign-out redirect URI below.
```

Reply:
updated